### ISSUE TITLE
How to translate the labels from locallang_hooks.xml via TypoScript?

### ISSUE THREAD

**Issue by typo3ua** [giselew/toctoc_comments#4]
I need to translate the labels from:
```
locallang_hooks.xml
locallang_ajax.xml
locallang_cedb.xml
...and others not from this file "EXT:toctoc_comments/pi1/locallang.xml" 
```
How to translate the labels from those locallang_****.xml via TypoScript?
